bundles/io/org.eclipse.smarthome.io.security/about.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
```

Comment:
Please replace this file with a `NOTICE` file, because with the change to EPL-2 we are no longer using `about.html` files, see https://github.com/eclipse/smarthome/blob/master/bundles/core/org.eclipse.smarthome.core/NOTICE for an example.